features/wallet/impl/src/main/java/com/tangem/feature/wallet/presentation/wallet/state/factory/WalletStateFactory.kt

```diff
@@ -102,17 +99,32 @@ internal class WalletStateFactory(
         }
     }
 
-    fun getStateWithOpenBottomSheet(content: WalletBottomSheetConfig.BottomSheetContentConfig): WalletStateHolder {
+    fun getStateWithOpenBottomSheet(
```

Comment:
Кажется так понятнее
```suggestion
    fun getStateWithWalletBottomSheet(
```

Reply:
изменил